src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1367,4 +1356,16 @@ private void OnSpeciesNameTextChanged(string newText)
             speciesNameEdit.Set("custom_colors/font_color", new Color(1, 1, 1));
         }
     }
+
+    private Dictionary<string, float> SortBarData(Dictionary<string, float> bar)
+    {
+        Dictionary<string, float> result = bar;
+
+        result = result.OrderBy(
+            i => (i.Key != "osmoregulation" && i.Key != "baseMovement") ? i.Key : string.Empty)
```

Comment:
This still isn't a complete sort function. I linked previously a stackoverflow answer that shows an entire. I couldn't find that again, so here's another one: https://stackoverflow.com/questions/43345685/c-sharp-custom-sort-string-icomparer
see how you need to make sure that you check all conditions both ways. In C++ not doing that will corrupt the heap.